resources/repos.json

```diff
@@ -770,6 +770,28 @@
 			}
 		}
 	},
+	"Seggan/Slimy-Power-Suits:master": {
+		"sonar": {
+			"enabled": false
+		},
+		"options": {
+			"prefix": "DEV"
+		},
+		"dependencies": {
+			"Minecraft Version(s)": {
+				"1": "1.15.x, 1.16.x"
+			},
+			"CS-CoreLib Version": {
+				"1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/CS-CoreLib/master/#90\">dev #90 or higher</a>"
+			},
+			"Slimefun Version": {
+				"1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#16\">RC-16 or higher</a>"
+			},
+			"LiteXpansion Version": {
+				"1": "<a class=\"link_info\" href=\"builds/J3fftw1/LiteXpansion/master/#73\">dev #73</a>
```

Comment:
Missing quotation marks at the end of this line.
Also why does this depend on LX?

Reply:
Its an addon for LiteXpansion. It uses LX items.